packages/turf-clone/index.js

```diff
@@ -70,12 +77,21 @@ function cloneFeature(feature) {
  * @returns {FeatureCollection<any>} cloned Feature Collection
  */
 function cloneFeatureCollection(geojson) {
-    return {
-        type: 'FeatureCollection',
-        features: geojson.features.map(function (feature) {
-            return cloneFeature(feature);
-        })
-    };
+    const cloned = {type: 'FeatureCollection'};
+    // Add id & bbox first
+    if (geojson.id) cloned.id = geojson.id;
+    if (geojson.bbox) cloned.bbox = geojson.bbox;
+
+    // Custom Properties
+    Object.keys(geojson).forEach(function (key) {
```

Comment:
I was able to preserve the `properties` using the default setting, so I've dropped that param. Doesn't really have any backwards compatibility effects since it behaves the same way if `true` or `false`.